libr/core/agraph.c

```diff
@@ -4046,6 +4060,106 @@ static bool toggle_bb(RCore *core, ut64 addr) {
 	return false;
 }
 
+typedef struct r_canvas_location {
+	int x;
+	int y;
+} RPosition;
+
+static const char *strstr_xy(const char *p, const char *s, int *x, int *y) {
+	const char *d = strstr (p, s);
+	if (!d) {
+		return NULL;
+	}
+	const char *q;
+	int nl = 0;
+	int nc = *x;
+	for (q = p; q < d; q++) {
+		if (*q == '\n') {
+			nl++;
+			*x = 0;
+			nc = 0;
+		} else {
+			nc++;
+		}
+	}
+	*x += nc;
+	*y += nl;
+	return d;
+}
+
+static char *old_word = NULL;
```

Comment:
put this somewhere else, not static. Maybe in RAGraph?

Reply:
yes i dont want globals